.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,46 @@
+<!--- Provide a general summary of your changes in the Title above -->
+
+## Description
+
+<!--- Describe your changes in detail -->
+
+## Related Issue
+
+<!--- Every pull request must have a reference to a GitHub or Adobe internal issue. -->
+<!--- If suggesting a new feature or change, please discuss it in an issue first -->
+<!--- If fixing a bug, there should be an issue describing it with steps to reproduce -->
+<!--- Please link to the issue here: -->
+
+## Motivation and Context
```

Comment:
This could (should?) be part of the issue.

Reply:
I would say that the issue context and motivation is clear. I guess opening a PR should basically be linked to some issue / feature .